src/components/menu/menu.tsx

```diff
@@ -133,6 +140,10 @@ export class Menu {
                         style={cssProperties}
                     >
                         <limel-list
+                            class={{
```

Comment:
@jgroth said:
I just had a quick look and I think it would be good if we could make the interface a bit more consistent. At the moment you enable it by setting a class for the list, but you set a prop if you want to use it for the menu. I think it would be better if we pick one method and use it in both places.
_____

Thanks Johan

We had a discussion about this subject (using props or classes) here: https://github.com/Lundalogik/lime-elements/pull/1281#discussion_r690332629

I personally prefer props. I feel it's easier for developers to deal with them. But I don't have strong opinion. 

However, here in this PR two things have happened.
One is that in the list, we add a class to render the items in a grid layout. This is to keep it consistent with the other styling classes in list component.

In the example in limel-list, there are two classes (`has-grid-layout` & `has-interactive-items`) used simultaneously but they can also be used individually. 
![image](https://user-images.githubusercontent.com/35954987/129862371-4966c156-ce30-4cda-b788-a150c830f1ba.png)

However, in the case of `limel-menu`, I actually need both of these classes to be applied in the same time to avoid the flat look and feel. This is why I used the `prop` instead. I didn't want to leave the responsibility to the consumer and also add extra notes about this that they have to have both classes. 

(note that `has-interactive-items` makes the items look more clickable and more interactive)

Also I don't think it is possible to add a class and affect a nested component in the menu. In the same time, I wanted to use the styling CSS variables of the new grid layout in limel-list, even in the menu, and they must be sent to portal and so forth... So it'd would be impossible to do it without the prop :)

Reply:
@Kiarokh If you prefer props, then use props. You're the one who started using helper-classes, and no one else in the team has ever added any, as far as I know 😄

Reply:
😄 that's right. Well, it was when I knew nothing about TypeScript and how things worked. Helper classes sounded like the easiest and least independent way of doing things. But now I started getting this feeling that props are perhaps easier for our consumers. Still somehow immature thoughts. So if others agree as well, for upcoming ones we can go for props.